src/libraries/Microsoft.Win32.Registry/README.md

```diff
@@ -0,0 +1,15 @@
+# Microsoft.Win32.Registry
+
+This assembly provides APIs for accessing and modifying the Windows Registry.
+
+## Contribution Bar
+
+- [x] [We only consider fixes to maintain or improve quality](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [x] [We consider PRs with refactoring changes due to new language features](../README.md#secondary-bars)
```

Comment:
I am not confident we want this contribution bar for this one. The last version where we published it was in 5.0: https://www.nuget.org/packages/Microsoft.Win32.Registry 

@danmoseley what would you say should be our bar for an assembly like this?

Reply:
Is the issue with the contribution bar about accepting refactoring changes?

Reply:
Bump. Do we want to accept refactoring changes for this package? Is this something we can fix later on?
/cc @carlossanlop @danmoseley

Reply:
@ViktorHofer @ericstj would you agree that this is the right bar for this assembly?

Reply:
We just recently accepted a refactoring change in Registry, so I would say yes, we do.

Reply:
> The last version where we published it was in 5.0: https://www.nuget.org/packages/Microsoft.Win32.Registry

@carlossanlop it became inbox in 6.0.